components/composed/ConfirmModalBody/ConfirmModalBody.styles.ts

```diff
@@ -0,0 +1,24 @@
+import styled, { css } from "styled-components";
+import { basePadding } from "theme/mixins/appearance";
+import { respond } from "theme/mixins/base";
+import Button from "components/atomic/buttons/Button";
+import { pxToRem } from "theme/mixins/functions";
+
+export const ButtonWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  padding-block-start: ${basePadding(10)};
+  gap: 15px;
+
+  ${respond(
+    css`
+      flex-direction: column;
+    `,
+    50
+  )}
+`;
+
+export const ConfirmButton = styled(Button)`
+  width: ${pxToRem(197)};
+  max-width: 90%;
```

Comment:
Instead of setting a strict width on the buttons, set these to be `flex: 1 1 auto`. Then the buttons will fill their container, and the gap can be adjusted as needed! You could even take away the media query and set `flex: wrap;` on the parent, and let the browser decide when the buttons need to stack.